modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -40,6 +40,7 @@
       <component
         v-show="displayComponent(field)"
         v-model="fieldState[field.name]"
+        :class="{ 'apos-field__wrapper--highlight': meta[field.name]?.['@apostrophecms/schema:highlight'] }"
```

Comment:
we should make this line smaller than 90 characters, the linter will complain otherwise. 
Maybe using a computed prop.

Reply:
Done with a method (since there is a parameter to pass)